### PR TITLE
warning that backups don't work

### PR DESCRIPTION
**Pull request by beckettsean**
@rkuchan @gunnaraasen @rossmcdonald thoughts?

### CODE REVIEW

File: content/influxdb/v0.9/administration/backup_and_restore.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ menu:
 
 Starting with v0.9.0, InfluxDB has the ability to snapshot a single data node at a point-in-time and restore it.
```
Comment (gunnaraasen) [inline content/influxdb/v0.9/administration/backup_and_restore.md]: Might want to just remove this line.

**Comment by gunnaraasen** [comment]
This is good for now. Though we should probably replace this whole page with an explanation of how to make a full directory backup instead of using the built-in snapshot functionality.

**Comment by rkuchan** [comment]
I was a bit confused by "significant subset of users" - is it just everyone who can't use it or is it people on specific versions?

**Comment by beckettsean** [comment]
@rkuchan basically I'm shocked whenever it works for anyone, but that sounds very pessimistic. I will restructure the page to give the file system backup as @gunnaraasen suggests and then leave the other stuff as "experimental" at the bottom.

**Comment by rkuchan** [comment]
Fixes https://github.com/influxdata/docs.influxdata.com/issues/24

**Comment by beckettsean** [comment]
merge hell, closing in favor of https://github.com/influxdata/docs.influxdata.com/pull/195

**Pull request closed without merging**